GeneralsMD/Code/GameEngine/Source/GameClient/GUI/GUICallbacks/Diplomacy.cpp

```diff
@@ -169,6 +169,8 @@ BriefingList* GetBriefingTextList(void)
 //-------------------------------------------------------------------------------------------------
 void UpdateDiplomacyBriefingText(AsciiString newText, Bool clear)
 {
+	if (TheWindowManager == NULL)
```

Comment:
There are a lot of TheWindowManager null checks. Can we perhaps have a Null implementation for the window manager and avoid all these null checks?